Whois integration connectivity issue (#14519)
demisto/content: Packs/Whois/ReleaseNotes/1_2_5.md

#### Integrations
##### WhoisFix

> - Fixed an issue where **domain** command would fail with indian domains.

Reviewer: ```suggestion
- Fixed an issue where ***domain*** command would fail on *.in* domains.
```
